### PR TITLE
work torwards using psycopg2

### PR DESCRIPTION
**Pull request by mtippmann**
still work in progress... 

- added a `INSTALL.md` with some notes on setting this up with virtualenv 
- added `requirements.txt` for use with pip/virtualenv 
- minor changes to `database.py` seems to work now 
- gnupg from pip seems to use homedir instead of gnupghome

### CODE REVIEW

**Comment by aparcar** [comment]
Sorry I messed up here, please reopen the PR

**Comment by mtippmann** [comment]
Thanks for the review, I'll update and repoen!

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Installation Notes
+
+## Prerequisites
+
+- Debian/Ubuntu
```
[the remark this answers is not in the document]
Comment (mtippmann) [reply INSTALL.md]: I'll add seperate sections for OpenWrt and Ubuntu/Debian

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+
+### 0. Install required packages 
+
+    # apt install python3-dev python3-venv build-essential cmake git postgresql-10 
+    
+    
+### 1. Clone Repository 
+
+    $ git clone https://github.com/weimarnetz/attendedsysupgrade-server.git
```
[the remark this answers is not in the document]
Comment (mtippmann) [reply INSTALL.md]: Sorry, totally missed that.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
+click==6.7
+Flask==1.0.2
+gnupg==2.3.1
```
[the remark this answers is not in the document]
Comment (mtippmann) [reply requirements.txt]: Thanks for the hint, this explains my problems. Again Sorry

**Comment by aparcar** [comment]
@mtippmann ping :)?

**Pull request merged**